src/dialog/dlgreplacecuecolordlg.ui

```diff
@@ -111,9 +98,41 @@
         </item>
        </widget>
       </item>
+      <item row="2" column="0" colspan="3">
+       <widget class="QLabel" name="labelReplaceAllColors">
+        <property name="lineWidth">
+         <number>1</number>
+        </property>
+        <property name="text">
+         <string>You didn't specify any conditions. This means that the colors of all cue points in the library will be replaced.</string>
```

Comment:
Somehow the text does not show for me. I see the "... by" label below moving slightly up and down indicating the widget is being shown/hidden when I uncheck/check the criteria checkboxes, but I never see this text.

Reply:
a01675b gets the warning icon to show, but not the text
![image](https://user-images.githubusercontent.com/9455094/79135563-28f4d100-7d75-11ea-9091-c13ccf7111e1.png)

Reply:
Hmm, for me the I on is null, maybe need to use a separate label for the text.

Reply:
With d256c6b the gap between the icon and the text is a bit odd but it works. 
![image](https://user-images.githubusercontent.com/9455094/79137979-55125100-7d79-11ea-9be6-2624ba4cd581.png)

Reply:
The text is a bit verbose. How about, "All hotcues' colors will be changed if you do not specify any conditions."

Reply:
I went with "If you don't specify any conditions, the colors of all cues in the library will be replaced."

Reply:
Yes, that's nice and succinct.